Chandan/chandan_practice.c

```diff
@@ -1,106 +1,132 @@
-#include <stdio.h>
-#include <time.h>
-#include <math.h>
-
-#define SECONDS_PER_MINUTE 60
-#define MINUTES_PER_HOUR 60
-#define HOURS_PER_DAY 24
-#define PROMPT_STRING "Type any key to show time, q to show time then quit: "
-
-#define TIME_TYPE unsigned long 
-
-char getChar() ;
-void updateTimes() ;
-void initTimes() ;
-void printPrompt() ;
-void printTime (TIME_TYPE timeInit) ;
-
-TIME_TYPE start, end;
-
+#include<stdio.h>
+#include<math.h>
 int hello_c()
 {
-        char command = 0;
-        initTimes();
-        delay();
-        updateTimes();
-        printTime(end-start);
-        return 0;
-}
-
-int getstarttime()
-{
-    int starttime;
-    starttime = 1;
-    return starttime;
-}
-
-int getendtime()
-{
-    int endtime;
-    endtime = 32767;
-    return endtime;
-}
-
-int delay()
-{
-   int a = getstarttime();
-   int b = getendtime();
-   int c = a;
-   int d = b;
-   
-   for (c = 1; c <= 32767; c++)
-       for (d = 1; d <= 32767; d++)
-       {}
+   float a = 14.5;
+   float b = 16;
+   float c = 17.5;
+   int x = 3;
+   int y = 6;
+   int z = 5;
+   int i, HCF, max, step, LCM;
+   float sum, prod, divs, mod, modu, cira, cirb;
+   float sub, diff, divi, s, area, cir, rec;
+   printf("\n The values are A = %f, B = %f, C=%f \n", a,b,c );
+   printf(" \n The values of X = %d, Y = %d, Z= %d \n", x,y,z);
+   
+   printf("\n The sum of all the numbers");
+   sum = a + b + c;
+   printf(" \n sum = %f \n", sum);
+   
+   printf("\n The difference of B and A ");
+   sub = b - a;
+   printf("\n difference = %f", sub);
+   
+   printf("\n The difference of C and B");
+   diff = c - b;
+   printf("\n difference = %f \n ", diff);
+   
+   printf(" \n The product of all numbers ");
+   prod = a * b * c;
+   printf("\n the product = %f \n", prod);
+   
+   printf("\n The division of B and A ");
+   divs = b/a;
+   printf(" \n The quotient = %f  \n", divs);
+   
+   printf("\n The division of C and B ");
+   divi = c/b;
+   printf("\n The quotient = %f \n", divi); 
+   
+   printf(" \n The even and Odd numbers of X, Y and Z :");
+   if( x%2 == 0)
+   printf ("\n X is an even number which is: %d ", x);
+   else
+   printf("\n X is an odd number which is : %d ", x);
+   
+   if( y%2 == 0)
+   printf ("\n Y is an even number which is: %d", y);
+   else
+   printf("\n Y is an odd number which is : %d", y);
+   
+   if( z%2 == 0)

```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Condition 'z%2==0' is always false](https://app.codacy.com/manual/e-mock/CProgramming-Team6/pullRequest?prid=5572102)